lib/tasks/jenkins.rake

```diff
@@ -1,25 +1,39 @@
 begin
   require "ci/reporter/rake/minitest"
 
-  namespace :foreman do
-    task :set_test_runner do
-      ENV['TESTOPTS'] = "#{ENV['TESTOPTS']} #{Rails.root}/test/test_runner.rb"
+  namespace :jenkins do
+    task :unit do
+      system "rake db:migrate RAILS_ENV=test"
```

Comment:
I'm doing this on a separate because otherwise the next task (rake test) doesn't have the db initialized when `validates_lengths_from_database` is called on models. This causes errors on the first run, so Jenkins would set the build as failed even if normally tests pass.